### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by menalq**

### CODE REVIEW

**Comment by menalq** [comment]
@brandonsavage: Please review my homework for lesson 1 :)

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "ciprianmatis/masterclass-repo",
+    "description": "Masterclass",
+    "require-dev": {
+        "phpunit/phpunit": "~4.7@dev"
```
Comment (brandonsavage) [inline composer.json]: You'll want to make sure and track a stable version of packages. 4.5 is current.

File: src/Controller/Comment.php [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <?php
+Namespace MOOP\Controller;
+
+Use PDO;
```
Comment (brandonsavage) [inline src/Controller/Comment.php:4]: Typical coding styles don't capitalize use, but it's personal style.

File: src/Controller/User.php [hunk 3]
```diff
@@ -1,4 +1,7 @@
 <?php
+Namespace MOOP\Controller;
```
Comment (brandonsavage) [inline src/Controller/User.php:2]: Same thing on namespace and capitalization.

**Comment by brandonsavage** [comment]
Good work!

**Comment by menalq** [comment]
Thank you, I'll do the suggested changes, commit and merge the branch

**Pull request merged**